cli/objects/rule.py

```diff
@@ -274,6 +276,68 @@ def _get_jira_additional_labels(
         else:
             return None
 
+    def _get_jira_assignee(self, rule_dict: dict[Any, Any]) -> Optional[str]:
+        """
+        Determines if a Jira Assignee is defined in a rule. If it is, validate it and return the string.
+
+        :param rule_dict: A dictionary object representing a user-defined firewatch rule.
+        :return: A string of the Jira assignee to use in a firewatch rule. If one is not defined, return None
+        """
+        if "jira_assignee" in rule_dict.keys():
+            try:
+                jira_assignee = rule_dict["jira_assignee"]
+            except Exception as ex:
+                self.logger.error(ex)
+                exit(1)
+            if isinstance(jira_assignee, str):
+                if "@" in jira_assignee:
```

Comment:
You can check for `"@redhat.com"` here

Reply:
@Simranpal I considered this, but wasn't sure if we have users in Jira that have different domains in their email (IBM maybe?)

Reply:
@Simranpal how is this: [c56a85f](https://github.com/CSPI-QE/firewatch/pull/23/commits/c56a85fed24aa9c19101f22d6ecc5f19695ec4d4)

Reply:
yes looks good @calebevans